config/samples/resources/backupdrbackupplanassociation/dependencies.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeInstance
+metadata:
+  name: backupdrbackupplanassociation-dep
+spec:
+  machineType: n1-standard-1
+  zone: us-central1-a
+  bootDisk:
+    initializeParams:
+      sourceImageRef:
+        external: debian-cloud/debian-11
+  networkInterface:
+  - networkRef:
+      external: backupdrbackupplanassociation-dep
+---
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeDisk
+metadata:
+  name: computedisk-dep
+spec:
+  location: us-west1-a
+  imageRef:
+    external: debian-cloud/debian-11
+---
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeImage
```

Comment:
Does this work against GCP?  I think this API is quite hard to use, so ... awesome if so!